ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py

```diff
@@ -193,22 +238,25 @@ def computeLimit(value, ccurve_TOTAL_READS):
         else:
             first_point = middle_point +1
         iterations += 1
-        
+
     return first_point
 
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser("plot_complexity_curves.py", description=plot_complexity_curves.__doc__)
-    parser.add_argument('ccurves', metavar='<preseq file>', nargs='+', 
+    parser.add_argument('ccurves', metavar='<preseq file>', nargs='+',
         help="List of input files generated by preseq")
-    parser.add_argument('-r', '--real-counts', dest='real_counts_path', type=str, default=False, 
+    parser.add_argument('-c', '--coverage', dest='coverage', type=int, default=0,
+        help="Use coverage on axes instead of read counts. Enter number of base pairs of refernce. Human GRCh38.p2 = 3221487035, GRCh37 = 3137144693")
+    parser.add_argument('-l', '--read-length', dest='read_length', type=int, default=0,
+        help="Sequence read length, for use in coverage calculations")
+    parser.add_argument('-r', '--real-counts', dest='real_counts_path', type=str, default=False,
         help="File name for file with three columns - BAM filename, total number reads, number of unique reads (space delimited)")
-    parser.add_argument('-o', '--output-name', dest='output_name', type=str, default='complexity_curves', 
+    parser.add_argument('-o', '--output-name', dest='output_name', type=str, default='complexity_curves',
         help="Output name (.png will be automatically added)")
-    parser.add_argument('-m', '--x-min', dest='x_min', type=int, default=0, 
+    parser.add_argument('-m', '--x-min', dest='x_min', type=int, default=0,
         help="Lower x-limit (default 0)")
-    parser.add_argument('-x', '--x-max', dest='x_max', type=int, default=500000000, 
+    parser.add_argument('-x', '--x-max', dest='x_max', type=int, default=500000000,
         help="Upper x-limit (default 500 million)")
```

Comment:
Could do, or just make the default have the number of reads that corresponds to 60X for human..